composer.json

```diff
@@ -13,7 +13,7 @@
         "php": ">=7.4 || ^8.0",
         "nette/neon": "^3.2",
         "symfony/polyfill-php80": "^1.26",
-        "thecodingmachine/safe": "^1.3"
+        "thecodingmachine/safe": "^1.3 || ^2.2"
```

Comment:
Any particluar reason to add only 2.2+ and not 2.1 or 2.0 or similar?

Reply:
not really, 2.0 should be sufficient, I'll change it.